### PR TITLE
REDDEER-1875 Extend ErrorCollector by catching a screenshot

### PR DESCRIPTION
**Pull request by luvalach**
REDDEER-1875 Extend ErrorCollector by catching a screenshot

Signed-off-by: Lukáš Valach <lvalach@redhat.com>

### CODE REVIEW

**Comment by genie-reddeer** [comment]
Can one of the admins verify this patch?

**Comment by odockal** [comment]
testPR

**Comment by genie-reddeer** [comment]
Can one of the admins verify this patch?

File: plugins/org.eclipse.reddeer.junit/src/org/eclipse/reddeer/junit/rules/ErrorCollector.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ * The ErrorCollector rule allows execution of a test to continue after the
+ * first problem is found. See {@link org.junit.rules.ErrorCollector} for detail
+ * description. This subclass adds automatic screenshot capturing.
+ * 
+ * @see org.junit.rules.ErrorCollector
+ * 
+ * @author lvalach
+ *
+ */
+public class ErrorCollector extends org.junit.rules.ErrorCollector {
```
Comment (rawagner) [inline plugins/org.eclipse.reddeer.junit/src/org/eclipse/reddeer/junit/rules/ErrorCollector.java]: I would call it ErrorCollectorWithScreenshot. @apodhrad  what do you think ?

**Comment by luvalach** [comment]
I renamed ErrorCollector to ErrorCollectorWithScreenshot and also fixed problem in test. Could you run tests again please?

**Comment by rawagner** [comment]
testPR

**Comment by luvalach** [comment]
It seems to be ok.

**Pull request merged**